ansible/www-standalone/resources/scripts/build-site.sh

```diff
@@ -17,7 +17,7 @@ clonedir=/home/www/github/${site}
 
 if [ ! -d "${clonedir}" ]; then
   repo="${site}.org"
-  git clone https://github.com/nodejs/${repo}.git $clonedir
+  git clone --depth 2 https://github.com/nodejs/${repo}.git $clonedir
```

Comment:
I'm not sure this is necessary -- we only clone if the directory does not exist and AFAIK we haven't wiped it (we reset, clean, fetch and checkout but don't reclone in the usual triggering of this script.).

Reply:
Gotcha. But would this be useful for new clones? I'd say so?